proto/priest.proto

```diff
@@ -45,15 +45,20 @@ message ShadowPriest {
     message Rotation {
       enum RotationType {
           Unknown = 0;
-          Standard = 1;
+          Basic = 1;
+		  ClipAlways = 2;
+		  IntelligentClipping = 3;
       }
-      RotationType type = 1;
+	  
+      RotationType rotation_type = 1;
+	  bool use_dev_plague = 3; // only undead
```

Comment:
I think symbol of hope and starshards are worth including too?

Reply:
its on the TODO in the priest file. I figure we can add that once we are happy with the main rotation. We have been using dwarf for most the testing.